storage/innobase/include/log0online.h

```diff
@@ -54,6 +54,13 @@ changed page bitmap which is then written to disk.
 @return true if log tracking succeeded, false if bitmap write I/O error */
 bool log_online_follow_redo_log(void);
 
+/** Read and parse redo log past the last checkpoint lsn for FLUSH
+CHANGED_PAGE_BITMAPS command
+
+@return true if log tracking succeeded, false if bitmap write I/O error
+*/
+bool log_online_follow_redo_log_past_checkpoint_lsn(void);
```

Comment:
This is the main entry point into the changed page tracking - IMHO it should have a short name, such as original log_online_follow_redo_log, and that one should be renamed instead (i.e. log_online_follow_redo_log_one_pass or something else.)

Reply:
makes sense, will rename